raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -2132,15 +2132,30 @@ private long pollLeader(long currentTimeMs) {
             currentTimeMs
         );
 
-        long timeUntilSend = maybeSendRequests(
-            currentTimeMs,
-            partitionState
-                .lastVoterSet()
-                .voterNodes(state.nonAcknowledgingVoters().stream(), channel.listenerName()),
-            this::buildBeginQuorumEpochRequest
-        );
+        long timeUntilNextBeginQuorumSend = state.timeUntilBeginQuorumEpochTimerExpires(currentTimeMs);
+
+        if (timeUntilNextBeginQuorumSend == 0) {
+            System.out.println("timeUntilNextBeginQuorumSend = 0");
```

Comment:
`System.out.println` should be replaced by `logger`

Reply:
thanks, left a debugging message in by accident. I'll change this to trace to reduce log spamming